audit/projects/k8s-infra-e2e-boskos-gpu-01/services/compute/project-info.json

```diff
@@ -119,6 +119,10 @@
       "limit": 100,
       "metric": "SECURITY_POLICY_RULES"
     },
+    {
+      "limit": 1000,
+      "metric": "XPN_SERVICE_PROJECTS"
+    },
```

Comment:
Don't think this is anything we did.  Suspect this is a new quota type introduced for https://cloud.google.com/vpc/docs/shared-vpc